llvm/lib/Transforms/InstCombine/InstCombineCompares.cpp

```diff
@@ -3133,7 +3133,9 @@ Instruction *InstCombinerImpl::foldICmpAddConstant(ICmpInst &Cmp,
 
   if (ICmpInst::isUnsigned(Pred) && Add->hasNoSignedWrap() &&
       C.isNonNegative() && (C - *C2).isNonNegative() &&
-      computeConstantRange(X, /*ForSigned=*/true).add(*C2).isAllNonNegative())
+      computeConstantRange(X, Add, /*ForSigned=*/true)
```

Comment:
I think `Add` is getting converted to a bool here and being used for `ForSigned` (then `/*ForSigned=*/true` is being used for `UseInstrInfo`).

Reply:
Members take precedence over functions from an outer namespace, but I could use a different name for the member to be more clear.